lib/Alembic/Ogawa/IStreams.cpp

```diff
@@ -38,7 +38,8 @@
 #include <stdexcept>
 
 
-#if defined (__unix__) || (defined (__APPLE__) && defined (__MACH__))
+#if defined (__unix__) defined (__HAIKU__) \
```

Comment:
defined (__unix__) defined (__HAIKU__)  is that an implicit &&?

Reply:
My mistake :)